code/modules/hydroponics/hydroponics.dm

```diff
@@ -521,6 +521,12 @@
 		mutmod = 0
 		adjustNutri(round(S.get_reagent_amount("robustharvestnutriment") *1 ))
 
+
+	//Fish Water is both an excellent fertilizer and waters
+	if(S.has_reagent("fishwater", 1))
```

Comment:
This is an absolutely horrendous if chain. Hilarious, considering how much you complained about this same type of thing in the past.